alias-forwarder-email-service/README.md

```diff
@@ -0,0 +1,98 @@
+- is the current proposal for the structure of the email address fine? Should we include something else? Should we exclude something?
+
+## Some proposals
+
+Here are some proposals for satisfying our goal.
+These are by no means to be considered the *best* proposals for any definition of "best". These are just proposals.
+
+### Proposal #1: an email address per customer
+
+This proposal merges the concepts of:
+-
+- having a dedicated email address for each customer
+- using the `+` sign trick to correctly redirect requests
+
+Let's consider customer `shiba`. We can create a dedicated email address `shiba@giantswarm.io` and then use the `deliveredto` Gmail filter to filter emails out. Some examples could be:
+
+- `deliveredto:"shiba+urgent+aws@giantswarm.io"`
+- `deliveredto:"shiba+support+apps@giantswarm.io"` (or *packs* :) )
+
+#### Pros
+
+- when a customer joins/leaves we can simply create/remove the email address
+- "low" price? One inbox per customer
+- no alias logic involved: this is a real email address
+
+#### Cons
+
+- we have to manage the different combinations and one inbox per customer
+
+### Proposal #2: just two email addresses
+
+This proposal relies on using the `+` sign trick to correctly redirect requests.
+
+We can define two email addresses: `urgent@giantswarm.io` and `support@giantswarm.io`.
+
+Let's consider customer `shiba`. We can discern between their requests depending on the additional information present after the first `+` sign. Some examples could be:
+
+- `deliveredto:"urgent+shiba+aws@giantswarm.io"`
+- `deliveredto:"support+shiba+apps@giantswarm.io"` (or *packs* :) )
+
+#### Pros
+
+- no additional inboxes with respect to now
+- no alias logic involved: these are a real email addresses
+
+#### Cons
+
+- we have to manage all the different combinations in the same inbox
+- when a customer leaves, we may need to go and modify some rules rather than simply deleting an inbox
+  - admittedly, we think and hope there won't be that many customers leaving us
+
+## Questions
+
+The following questions arise for both proposals:
+
+- having the customer in the email is not really useful at the moment. Should we still go for it "just in case"?
+  - it doesn't hurt
+- can we choose the action to be taken (forward to slack/opsgenie) depending on the `deliveredto` address?
+  - YES, we can!
+- can the various processes be automated somehow? Maybe using a [Google Apps Script](https://script.google.com/)?
+  - (scenario 1) email address creation/deletion
+  - (both scenarios) "routes": support to slack vs. urgent to opsgenie; different areas
+  I haven't been able to find a sensible solution so far, but we can dig into this a bit more.
+
+# Final thoughts
+
+Considering the pros and cons of each solution, I believe we can proceed as follows:
+
+- Only use the support@giantswarm.io and urgent@giantswarm.io email addresses
+- Specify area and customer using the + symbol, e.g. support+adidas+kaas@giantswarm.io
```

Comment:
My main concern is to get customers to actually follow the pattern here.

Reply:
if they have some tooling where they set up the patterns it will work, I know of a customer using Remedy to notify us of an incident, so the actual engineer does not about the email address, (s)he just fill the form and submit

Reply:
@MarcelMue I think the idea is to give each customer only "their" email addresses. I wouldn't cross bridges before getting to them - we have time to improve.